rules/ethtool.mk

```diff
@@ -0,0 +1,12 @@
+# ethtool
+
+ETHTOOL_VERSION_BASE = 5.8
+export ETHTOOL_VERSION_BASE
+
+ETHTOOL = ethtool
+$(ETHTOOL)_SRC_PATH = $(SRC_PATH)/$(ETHTOOL)
+$(ETHTOOL)_PATH = $(BUSTER_FILES_PATH)/$(ETHTOOL)
+SONIC_MAKE_FILES += $(ETHTOOL)
+SONIC_COPY_FILES += $(ETHTOOL)
```

Comment:
@shlomibitton `SONIC_COPY_FILES += $(ETHTOOL)` and `SONIC_MAKE_FILES += $(ETHTOOL)` are mutually exclusive. You need `SONIC_MAKE_FILES += $(ETHTOOL)` only

Reply:
Fixed.